app.go

```diff
@@ -27,13 +30,33 @@ func NewApp() *App {
 // so we can call the runtime methods
 func (a *App) startup(ctx context.Context) {
 	a.ctx = ctx
+	fmt.Println("Backend debugging is working!")
 
 	godotenv.Load(".env")
 	if envKey := os.Getenv("POCKET_GPT_KEY"); envKey != "" {
 		POCKET_GPT_KEY = envKey
 	}
 }
 
+func (a *App) GetUserName() string {
+	curUser, err := user.Current()
+
+	if err != nil {
+		return ""
+	}
+
+	var name string
+
+	if curUser.Name != "" {
+		name = curUser.Name
+	} else {
+		split := strings.Split(curUser.Username, "/")
```

Comment:
Can you clarify what this is doing specifically and why the if check was needed? It's not clear why `curUser.Name` would be empty.